content/docs/studio/user-guide/teams.md

```diff
@@ -95,24 +96,46 @@ same as that of any other collaborator who has been assigned the `Admin` role.
 
 ## Manage your team and its views
 
-Once you have created the team, the team's page opens up.
+Once you have created the team, the team's workspace opens up.
 
-![](https://static.iterative.ai/img/studio/team_page.png)
+![](https://static.iterative.ai/img/studio/team_page_v3.png)
 
-On this page, you can perform three types of tasks:
+In this workspace, there are 2 pages - [Views](#views) and
+[Settings](#settings).
 
-- **Add a view.** Click on the `View` menu item to add views to the team's page.
-  The process for adding a view is the same as that for adding personal views
-  ([instructions](/doc/studio/user-guide/views/create-view)). However, the views
-  that you create within the team will be accessible to all members
-  (collaborators) of the team.
+### Views
 
-- **Edit collaborators.** You can click on the `Team` menu item to edit the
-  collaborators in the team.
+This is the views dashboard for the team. All the views on this dashboard are
+accessible to all members (collaborators) of the team.
 
-- **Change settings.** Finally, you can click on the `Settings` menu item to
-  change the team name, add credentials for the data remotes, and delete the
-  team. Note that these settings are applicable to the team and are thus
-  different from [view settings](/doc/studio/user-guide/views/view-settings).
+To add a view to this dashboard, click on `Add a view`. The process for adding a
+view is the same as that for adding personal views
+([instructions](/doc/studio/user-guide/views/create-view)).
 
-  ![](https://static.iterative.ai/img/studio/team_settings.png)
+### Settings
+
+In the team settings page, you can change the team name, add credentials for the
+data remotes, and delete the team. Note that these settings are applicable to
+the team and are thus different from
+[view settings](/doc/studio/user-guide/views/view-settings).
+
+Additionally, you can also
+[manage connections to self-hosted GitLab servers](#manage-connections-to-self-hosted-gitlab-servers)
+and [edit collaborators](#edit-collaborators).
+
+#### **Manage connections to self-hosted GitLab servers**
```

Comment:
Shouldn't need md bolding in headers. Need a change of styles?

Reply:
Since this is a 4th level heading, it's almost not distinguishable from the rest of the text. That's why I had to make it bold, so that it looks at least somewhat different (as shown below).

<img width="706" alt="image" src="https://user-images.githubusercontent.com/32855442/166194640-fef6bcb2-36a1-46e1-be2d-f120e0349f88.png">

Reply:
Need a change of styles? 🙂